services/beaconchain/src/main/java/tech/pegasys/artemis/services/beaconchain/BeaconChainController.java

```diff
@@ -184,8 +193,8 @@ private void onTick(Date date) {
           this.eventBus.post(new SlotEvent(nodeSlot));
           this.currentSlotGauge.set(nodeSlot.longValue());
           this.currentEpochGauge.set(compute_epoch_of_slot(nodeSlot).longValue());
-          STDOUT.log(Level.INFO, "******* Slot Event *******", ALogger.Color.WHITE);
-          STDOUT.log(Level.INFO, "Node slot:                             " + nodeSlot);
+          STDOUT.log(Level.DEBUG, "******* Slot Event *******", ALogger.Color.WHITE);
```

Comment:
Why the switch to slot event?

Reply:
switch to debug when printing*

Reply:
yep, should be INFO.  I had several other printouts that i needed to be DEBUG and i did a search and replace in the file and missed this.  Good catch